recipes/learnmsa/meta.yaml

```diff
@@ -1,48 +1,49 @@
 {% set name = "learnMSA" %}
-{% set version = "1.1.2" %}
+{% set version = "1.2.0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: b6b7362c967008d1dc2ed07045f05558ca2e6c1387c06054f75230a53f2b3ab2
+  sha256: f662a4541e6d374310899302d056492bf1b3e7013020bf93a1e6fda863a0dfde
 
 build:
   number: 0
   entry_points:
     - learnMSA = learnMSA.run:run_main
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
   noarch: python
-
+  
 requirements:
   host:
     - pip
     - python >=3.7
   run:
+    - libtiff
```

Comment:
Why as run dependency? Is this missing in a downstream dependency?

Reply:
Yes, but I don't know why it's suddenly missing. I do not directly depend on it.